server/static/tree/style.css

```diff
@@ -1,15 +1,13 @@
 body {
   background: url(texture-noise.png);
   overflow: auto;
-  margin: 0;
```

Comment:
This should stay to maximise space and may be causing the group dropdown to go to the next line?